packages/gensx-core/src/context.ts

```diff
@@ -1,5 +1,4 @@
-import { resolveDeep } from "./resolve.js";
-import { ComponentProps, Context, GsxComponent } from "./types.js";
+import { ComponentProps, Context } from "./types.js";
```

Comment:
is context an internal feature now or still a customer facing feature?

Reply:
Yes internal. Basically used for tracking state of parent/child nodes.